lambdas/src/MembershipSub-PromoCode-View-Dynamo-to-Salesforce.js

```diff
@@ -247,7 +245,7 @@ function makeAPICalls(csvData) {
 exports.handler = (event, context, callback) => {
 
     const TOUCHPOINT_BACKEND = /PROD$/.test(context.functionName) ? 'PROD' : /UAT$/.test(context.functionName) ? 'UAT' : 'DEV';
-    const TableName = `MembershipSub-PromoCode-View-${source}`;
```

Comment:
How did that ever work?!

Reply:
@paulbrown1982 I don't know to be honest. Maybe older node didn't check properly?

Reply:
The PROD code had drifted from the codebase! Either way, your correction is fine.

```
paul_brown ~/code/memsub-promotions (master) $ git diff
diff --git a/lambdas/src/MembershipSub-PromoCode-View-Dynamo-to-Salesforce.js b/lambdas/src/MembershipSub-PromoCode-View-Dynamo-to-Salesforce.js
index 2521050..9f2edff 100644
--- a/lambdas/src/MembershipSub-PromoCode-View-Dynamo-to-Salesforce.js
+++ b/lambdas/src/MembershipSub-PromoCode-View-Dynamo-to-Salesforce.js
@@ -246,7 +246,7 @@ function makeAPICalls(csvData) {
 
 exports.handler = (event, context, callback) => {
 
-    const TOUCHPOINT_BACKEND = /PROD$/.test(context.functionName) ? 'PROD' : /UAT$/.test(context.functionName) ? 'UAT' : 'DEV';
+    const source = /PROD$/.test(context.functionName) ? 'PROD' : 'UAT';
     const TableName = `MembershipSub-PromoCode-View-${source}`;
 
     docClient.scan({ TableName })
```